integration-libs/cdc/root/cdc-root.module.ts

```diff
@@ -47,6 +50,7 @@ export function defaultCdcComponentsConfig(): CmsConfig {
 }
 
 @NgModule({
+  imports: [CdcStoreModule],
```

Comment:
Why do we import `CdcStoreModule` in the root module? I guess we want it only in `CdcCoreModule`?

Reply:
noted.